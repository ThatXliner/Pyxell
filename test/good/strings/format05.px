
a = "A"
print "{{}}"
print "{{a}}."
print "{{{a}{a}}}"
print "{{{{}}{a}"
print "{a}{{{a}{{{{a}}}}a}}}}"
