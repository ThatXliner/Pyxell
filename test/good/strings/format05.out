{}
{a}.
{AA}
{{}A
A{A{{a}}a}}
